lmp/bb-build.sh

```diff
@@ -67,6 +67,13 @@ status "Run bitbake (save the global and image specific environments)"
 bitbake -e > ${archive}/bitbake_global_env.txt
 bitbake -e ${IMAGE} > ${archive}/bitbake_image_env.txt
 
+# Public LmP build and we are building the major release
+if [[ "${H_PROJECT}" == "lmp" ]] && [[ "${LMP_VERSION_MINOR}" == "0" ]] ; then
+    status "Run bitbake (generate mirror tarball)"
+    echo 'BB_GENERATE_MIRROR_TARBALLS = "1"' >> conf/local.conf
+    bitbake --runall fetch --no-setscene ${IMAGE}
+fi
```

Comment:
are we going to be saving this as a build artifact and if so, how expensive will this be?

Reply:
No, we will not save it as build artifacts for every build. The idea is to collect all the source archives need to build the public lmp when we do a release and upload them to the public https://storage.googleapis.com/lmp-cache/downloads/

The size of the _am62xx-evm_ machine is about 20G
```
du -sh downloads/
20G     downloads/
```

Reply:
The cleanup method can be a matter of remove the `/lmp-cache/downloads/` run a build for each active lmp release i.e: v92, v91, v90, v89, v88 etc and upload the worker downloads/ folder to the download mirror https://storage.googleapis.com/lmp-cache/downloads/

Reply:
so its for you running builds locally? looks good to me

Reply:
Not really, this PR is meant to be run on CI in the public lmp factory in each major release.

For example the workflow of the next release will be:

1. We tag the v93 on the lmp-manifest
2. The next build and only the next will create an archive on the bibake download folder of each git repo source code used in the build
3. At the end of the build we should also rsync the files on the downloads/ folder additionally to sstate-cache we already sync. The last step needs to be added for the existing script that rsync the sstate-cache

After that we will no longer depend on external sources code and we will have everything we need in our public mirror https://storage.googleapis.com/lmp-cache/downloads/